### DISCUSSION TITLE
'This setup requires the x64 .NET Runtime Version 7.0.0'

### DISCUSSION THREAD

**Discussion by mossuman** [JamesCJ60/Universal-x86-Tuning-Utility#144] [Q&A]
Forgive me as I am very new to all of this. 

Everytime I try to open the tool it displays this message. I have installed both dotnet sdk 7.0.403 and 7.0.0 in attempts to get it to work but the message still shows up. I feel a little daft having to ask for help before I've even got into the detailed stuff but hey.

Any assistance would be greatly appreciated.

**Comment by PeterSmoofwah** [comment]
I ran into the same issue, on the download page get the runtime not the sdk.
There's like 4 different similarly named downloads.

Then it'll ask you to get 7.0.0.14 or something so get the runtime for that too not the sdk

**Reply by onetakeseb** [reply]
Could you send a link to this I can't seem to find it?

**Reply by nikfed2018** [reply]
Get the desktop runtime, it's what you need
